### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by gnufied**
update readme file to be consistent with other CSI sidecars.

/assign @msau42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,65 @@
-# external-resizer
+[![Build Status](https://travis-ci.org/kubernetes-csi/external-resizer.svg?branch=master)](https://travis-ci.org/kubernetes-csi/external-resizer)
 
-This is an external Kubernetes controller which can expand volumes using CSI volume Drivers. It's under heavy development
-and of alpha quality.
+# CSI Resizer
 
-# Build
+The CSI `external-resizer` is a sidecar container that watches the Kubernetes API server for `PersistentVolumeClaim` object edits and
```
Comment (msau42) [inline README.md]: "object edits" => "updates"

Comment (gnufied) [reply README.md]: fixed

**Pull request merged**